### ISSUE TITLE
Endgamestats - Weldingtime

### ISSUE THREAD

**Issue by GhoulofGSG9** [NS2CDT/NS2Plus#8]
<a href="https://github.com/Handschuuh"><img src="https://avatars3.githubusercontent.com/u/38935291?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [Handschuuh](https://github.com/Handschuuh)**
_Thursday May 03, 2018 at 15:19 GMT_
_Originally opened as https://github.com/sclark39/NS2Plus/issues/87_

----

If you get no points for welding players/structures (which can be crucial) you need to have this score at the end of a game.

It needs to distinguish between welding players and structures - maybe even like the Accuracy

Welding 
0:52 ( 2:33 )

If you hover your mouse over the time or the "Welding" then you see a text

Welding players ( Weldingtime including structures)

Or just structures in the brackets...

**Comment by JSchriewer** [comment]
I agree and I would like to see gorge healing (healing aliens) as well. At least it is a more valuable information than building time.
